collector/textfile.go

```diff
@@ -77,6 +77,10 @@ func convertMetricFamily(metricFamily *dto.MetricFamily, ch chan<- prometheus.Me
 	}
 
 	for _, metric := range metricFamily.Metric {
+		if metric.TimestampMs != nil {
+			log.Warnf("Ignoring unsupported custom timestamp on textfile collector metric %v", metric)
```

Comment:
Let's make this `Debugf()`.

Reply:
I think it should be at least `Warnf()` because people might be relying on this currently, and if anyone does put timestamps into their textfiles, they should at least be warned that the output will not be what they expect.

Reply:
Hrm, I suppose.  But I thought the logging level philosophy was that we think about things from the exporter's point of view, and not the users's data point of view.  The exporter itself doesn't care about the timestamps.

Reply:
We do warn/error when something is setup incorrectly, such as out of order samples in Prometheus side. I think this should be a warning.

Reply:
👍

Reply:
Hm. On the other hand we even `log.Errorf()` when there's a read or parse error of the textfile. It can also be a user-caused problem, just even more severe than dropping timestamps. IMO we have to indicate *somehow* that something isn't working as the user would expect. For the full errors we also have a textfile error metric, but somehow I don't feel that dropping timestamps should trigger that, or that we should have a separate warning metric too. So IMO `Warnf()` is the best, but happy to be convinced otherwise.

Reply:
There is the option of considering files containing timestamps to be the same as files that don't parse. That's roughly what the pgw does in rejecting pushes with timestamps.